versiondb/tsrocksdb/opts.go

```diff
@@ -0,0 +1,63 @@
+package tsrocksdb
+
+import (
+	"runtime"
+
+	"github.com/linxGnu/grocksdb"
+)
+
+const VersionDBCFName = "versiondb"
+
+// NewVersionDBOpts returns the options used for the versiondb column family.
+// In the case of sstFileWriter, we don't enable dict compression, because the file writer won't report correct file size.
+func NewVersionDBOpts(sstFileWriter bool) *grocksdb.Options {
+	opts := grocksdb.NewDefaultOptions()
+	opts.SetCreateIfMissing(true)
+	opts.SetComparator(CreateTSComparator())
+	opts.IncreaseParallelism(runtime.NumCPU())
+	opts.OptimizeLevelStyleCompaction(512 * 1024 * 1024)
+	opts.SetTargetFileSizeMultiplier(2)
+	opts.SetLevelCompactionDynamicLevelBytes(true)
+
+	// block based table options
+	bbto := grocksdb.NewDefaultBlockBasedTableOptions()
+
+	// 1G block cache
+	bbto.SetBlockSize(32 * 1024)
+	bbto.SetBlockCache(grocksdb.NewClockCache(1<<30, 32*1024))
```

Comment:
Looks like the new rocksdb feature only support after v7.9 , need to update the build requirement in Cronos doc.

Reply:
I was trying to experiment with it, don't have much visible changes, the dump process is still mainly IO bound, I'll convert back to LRU cache first to make this PR simpler.